other_crates/bones_matchmaker/src/lib.rs

```diff
@@ -26,26 +22,28 @@ struct Config {
 }
 
 async fn server(args: Config) -> anyhow::Result<()> {
-    let task_pool = IoTaskPool::get();
+    let port = args.listen_addr.port();
 
-    // Generate certificate
-    let (cert, key) = certs::generate_self_signed_cert()?;
+    let secret_key = iroh_net::key::SecretKey::generate();
```

Comment:
Yeah, as far as I understand so far, it updates a centralized DNS record so that other nodes can resolve our node ID, so I think if we persist our key, then we can keep the same node ID and people will be able to find us, even on different a server/ip.

We should probably read the secret key from an environment variable, and I like the idea of generating if one isn't supplied, we should just log a warning to make it clear that it will be different every time the server is started.

Reply:
in the iroh main library we store the keys on disk in the same format as openssh does, this soul easy to add here, but could also use an env variable